lib/ash_authentication/add_ons/confirmation/transformer.ex

```diff
@@ -49,8 +49,8 @@ defmodule AshAuthentication.Strategy.Confirmation.Transformer do
       [_ | _] ->
         {:error,
          DslError.exception(
-           path: [:authentication, :strategies, :confirmation],
-           message: "Multiple confirmation strategies are not supported"
+           path: [:authentication, :add_ons, :confirmation],
+           message: "Multiple confirmation add ons are not supported"
```

Comment:
unrelated to this PR, but why not? Is it routing? If we put the add_on name in the route, we could support any number of confirmations, right?

Reply:
yeah, that occurred to me too. seems like an arbitrary decision I made at the start